schemas/post-hearing-comment.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$id": "#/properties/caseComments/items/anyOf/0",
+  "type": "object",
+  "title": "Create case comment request",
+  "description": "",
+  "default": {},
+  "required": [
+    "author",
+    "comment",
+    "hearingId"
+  ],
+  "properties": {
+    "author": {
+      "$id": "#/properties/caseComments/items/anyOf/0/properties/author",
+      "type": "string",
+      "title": "The author of the comment",
+      "default": ""
+    },
+    "comment": {
+      "$id": "#/properties/caseComments/items/anyOf/0/properties/comment",
+      "type": "string",
+      "title": "The comment",
+      "default": ""
+    },
+    "hearingId": {
+      "$id": "#/properties/caseComments/items/anyOf/0/properties/hearingId",
+      "type": "string",
+      "title": "The hearing id",
+      "default": ""
+    },
+    "createdByUuid": {
+      "$id": "#/properties/caseComments/items/anyOf/0/properties/createdByUuid",
+      "type": "string",
+      "title": "The created user uuid",
+      "default": ""
+    },
```

Comment:
I was wondering, would it be good to make the "createdByUuid" as a `required` parameter too?

Reply:
Fair shout, although @rschallapalli do we now need this at all? If it gets this from the user token then we can drop this entirely.

Reply:
No, we don't need any apart from author, comment, caseId

Reply:
Interesting - this schema defines the response, not the request. We're not checking that as such except for the Pact itself...

Reply:
It turns out that schema is serving as dual purpose for both the request and response with different fields appearing in each - I'll split them out to make the distinction clear